src/aks-preview/azext_aks_preview/_client_factory.py

```diff
@@ -10,6 +10,12 @@
 from azure.mgmt.msi import ManagedServiceIdentityClient
 from knack.util import CLIError
 
+# load container service client
+# another log
+# third log
+# more log
+# add more comments
+# still more comments
```

Comment:
Why not describe the details of those information？

Reply:
So it doesn't need to be merged, right? If so, please close it or label it as `do-not-merge`